### ISSUE TITLE
License ambiguity

### ISSUE THREAD

**Issue by pghmcfc** [briandfoy/module-extract-version#7]
The shipped LICENSE file is the Artistic 2.0 license but the Makefile.PL and VERSION.pm files say the license is same as perl (i.e. Artistic 1 or GPL v1 or later). Which is right?

**Comment by briandfoy** [comment]
It's Artistic 2.0. I've made the changes in f6b8632. Thanks.

**Comment by pghmcfc** [comment]
Just one thing left in 1.112; the LICENSE file itself starts with the text
```
  The Module::Extract::VERSION module is licensed under the same terms as
  perl itself, under the Artistic License 2.0.
```
I'd delete the "under the same terms as perl itself," bit, as perl is not licensed Artistic 2.0, it's GPL1 or later, or Artistic 1

**Comment by briandfoy** [comment]
Fixed in 56d9d69ebe1926ce458a006019adbc631ba12876.

Thanks,

**Comment by briandfoy** [comment]
This should be fixed in 1.113.Thanks!
